src/networking/Networking.ts

```diff
@@ -8,13 +8,9 @@ import { noop } from '../util/util';
 // The number of audio channels required by Discord
 const CHANNELS = 2;
 const TIMESTAMP_INC = (48000 / 100) * CHANNELS;
-const MAX_NONCE_SIZE = (2 ** 32) - 1;
+const MAX_NONCE_SIZE = 2 ** 32 - 1;
 
-const SUPPORTED_ENCRYPTION_MODES = [
-	'xsalsa20_poly1305_lite',
-	'xsalsa20_poly1305_suffix',
-	'xsalsa20_poly1305'
-];
+const SUPPORTED_ENCRYPTION_MODES = ['xsalsa20_poly1305_lite', 'xsalsa20_poly1305_suffix', 'xsalsa20_poly1305'];
```

Comment:
This one is probably best kept on multiple lines (add a `//` after the first `,` prettier will handle the rest

Reply:
I don't see why. There is no reason.